adhocracy4/comments_async/static/comments_async/comment.jsx

```diff
@@ -61,20 +61,18 @@ export default class Comment extends React.Component {
     this.state = {
       edit: false,
       showChildComments: false,
-      displayNotification: this.props.displayNotification,
       shorten: true,
       anchored: false,
       showModStatement: true,
       moderatorFeedback: this.props.moderatorFeedback
     }
 
-    setTimeout(
-      function () {
-        this.setState({ displayNotification: false })
-      }
-        .bind(this),
-      2000
-    )
+    if (this.props.displayNotification) {
```

Comment:
@hom3mad3 as I had to touch most bits of the (error) notifications I also found a solution for the success alert :)